dist/js/baltcogo-reporter.js

```diff
@@ -2,46 +2,48 @@
   'use strict';
 
   angular.module('baltcogoApp', [])
-    .config(["dataProvider", function (dataProvider) {
+    .config(["urlProvider", function (urlProvider) {
       var urls = [
         { "animal.breeds": '/mockups/citysourced/animal-breeds.json' },
         { "animal.colors": '/mockups/citysourced/animal-colors.json' },
         { "animal.types": '/mockups/citysourced/animal-types.json' },
         { "categories": '/mockups/citysourced/categories.json' },
-        { "pet.types": '/mockups/citysourced/pet-types.json' }
+        { "pet.types": '/mockups/citysourced/pet-types.json' },
+        { "report.create": "//testservices.baltimorecountymd.gov/api/baltcogo/createreport" },
+        { "report.get": "//testservices.baltimorecountymd.gov/api/citysourced/getreport/" },
+        { "reports.getByLatLng": "//testservices.baltimorecountymd.gov/api/citysourced/getreportsbylatlng"}
       ];
-      dataProvider.setUrls(urls);
+      urlProvider.setUrls(urls);
     }]);
 })();
 // http://www.learn-angular.org/#!/lessons/the-provider-recipe
-(function (app, querystringer) {
+(function (app) {
   'use strict';
 
-  function Data(arr) {
+  function Url(arr) {
     this.endpoints = {};
 
     for (var i = 0; i < arr.length; i++) {
       var obj = arr[i];
-      console.log(obj);
       var key = Object.keys(obj)[0];
 
       this.endpoints[key] = obj[key];
     }
   }
 
-  app.provider('data', function () {
+  app.provider('url', function () {
```

Comment:
Naming this **url** is still confusing. You're calling it urlProvider locally in the controller once it's injected, so why not call it urlProvider here?